external/Analyze.external.ruleset

```diff
@@ -0,0 +1,12 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!--
+  Copyright (c) Microsoft Corporation
+  SPDX-License-Identifier: MIT
+-->
+<RuleSet Name="External rules" Description="Override parent ruleset" ToolsVersion="16.0">
+  <Include Path="..\Analyze.default.ruleset" Action="Default" />
+  <Rules AnalyzerId="Microsoft.Analyzers.NativeCodeAnalysis" RuleNamespace="Microsoft.Rules.Native">
+      <!-- Set this to warning due to https://github.com/vbpf/ebpf-verifier/issues/239 -->
```

Comment:
can we file an issue to remove this once 239 is fixed?  I don't want to forget about this workaround, since the files in external should really get deleted then (the principle is that external should only have submodules under it), but a short term workaround is probably ok, hence not blocking this.

Reply:
Filed https://github.com/microsoft/ebpf-for-windows/issues/309 to track removing this override once upstream is fixed.